pkg/cli/cli_options.go

```diff
@@ -126,3 +128,19 @@ func OptionOrDefault(given string, defaultValue string) string {
 	}
 	return given
 }
+
+func ShouldCheckForUpdate(given string, defaultValue string, currVersion string) bool {
```

Comment:
small: can we rename these args? something like
```suggestion
func ShouldCheckForUpdate(updateStreamOverride string, defaultUpdateStream string, currVersion string) bool {
```

and then similarly below:

* `givenParts` => `streamOverrideParts`
* `givenVersion` => `streamOverrideVersion`

Reply:
yep can change for sure